requirements.txt

```diff
@@ -1,5 +1,5 @@
 huggingface_hub
-ultralytics==8.0.141
+ultralytics==8.0.124
```

Comment:
What is the reason for this version downgrade, any performance issues?

Reply:
No issues, just version we tested with